src/main.ts

```diff
@@ -0,0 +1,129 @@
+import os = require('os');
+
+import * as exec from '@actions/exec';
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import * as io from '@actions/io';
+import * as toolCache from '@actions/tool-cache';
+import * as rustCore from '@actions-rs/core';
+
+function getPlatformMatchingTarget(): string {
+    const platform = os.platform() as string;
+    switch (platform) {
+        case "linux":
+            return "x86_64-unknown-linux-gnu"
+        case "win32":
+            return "x86_64-pc-windows-msvc"
+        case "darwin":
+            return "x86_64-apple-darwin"
+        default:
+            throw new Error("Unsupported runner");
+    }
+}
+
+function optionIfValueProvided(option: string, value?: string): string {
+    return value ? ` ${option} ${value}` : '';
+}
+
+function getCheckReleaseArguments(): string[] {
+    return [
+        optionIfValueProvided('--package', rustCore.input.getInput('crate-name')),
+        optionIfValueProvided('--manifest-path', rustCore.input.getInput('manifest-path')),
+        rustCore.input.getInputBool('verbose') ? ' --verbose' : ''
+    ].filter(el => el != '');
+}
+
+function getGitHubToken(): string {
+    const token = process.env['GITHUB_TOKEN'] || rustCore.input.getInput('github-token');
+    if (!token)
+        throw new Error('Querying the GitHub API is possible only if the GitHub token is set.');
```

Comment:
Let's try to always use curly braces with statements like `if / for / while` etc.

Otherwise, we risk having [a `goto fail;` style bug](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) which would be really annoying to track down and deal with. The 1-character savings when not putting in the curly braces is extremely not worth the risk.

Reply:
Changed! I've also added the ESLint rule for this.